packages/editor-ui/src/components/ValueSurvey.vue

```diff
@@ -0,0 +1,182 @@
+				somewhat: "Thanks for your feedback! We'd love to understand how we can improve. Can we reach out?",
+				not: "Great to hear! Can we reach out to see how we can make n8n even better for you?",
+			},
+			VALUE_SURVEY_MODAL_KEY,
+		};
+	},
+	methods: {
+		closeDialog(): void {
+			if (!this.isEmailValid) {
+				this.$telemetry.track('User responded value survey score', { instance_id: this.$store.getters.instanceId, how_disappointed: '' });
+			}
+			this.resetForm();
+			this.$store.commit('ui/closeTopModal');
+		},
+		onInputChange(value: string): void {
+			this.isEmailValid = this.validateEmail(value);
+		},
+		resetForm() {
+			this.form.email = '';
+			this.form.value = '';
+		},
+		selectSurveyValue(value: string) {
+			this.form.value = value;
+			this.showButtons = false;
+			this.$store.dispatch('settings/submitValueSurvey', {value: this.form.value});
+			this.$telemetry.track('User responded value survey score', { instance_id: this.$store.getters.instanceId, how_disappointed: this.form.value });
+		},
+		send(): void {
+			if (this.isEmailValid) {
+				this.$store.dispatch('settings/submitValueSurvey', {email: this.form.email, value: this.form.value});
+				this.$telemetry.track('User responded value survey email', { instance_id: this.$store.getters.instanceId, email: this.form.email });
+				this.$showMessage({
+					title: 'Thanks for your feedback',
+					message: `If you’d like to help even more, answer this <a href="https://n8n-community.typeform.com/quicksurvey#how_disappointed=${this.form.value}&instance_id=${this.$store.getters.instanceId}">quick survey.</a>`,
+					type: 'success',
+				});
+			}
+			this.closeDialog();
+		},
+		validateEmail(email: string) {
+			const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+			return re.test(String(email).toLowerCase());
+		},
+	},
+});
+</script>
+
+<style module lang="scss">
+.title {
+	height: 16px;
+	font-size: var(--font-size-s);
+	line-height: 18px;
+	text-align: center;
+	color: var(--color-text-xlight);
+}
+
+.content {
+	display: flex;
+	justify-content: center;
+
+	.buttons {
+		button {
+			margin: 0 8px;
+			background-color: var(--color-background-xlight);
+			border: var(--color-background-xlight);
+			border-radius: 4px;
+			color: var(--color-background-dark);
+		}
+	}
+
+	.email {
+		.input {
+			display: flex;
+
+			input {
+				width: 290px;
+				height: 36px;
+			}
+
+			button {
+				margin-left: 10px;
+			}
+		}
+
+		.disclaimer {
+			margin-top: var(--spacing-4xs);
+			font-size: 10px;
+			color: var(--color-text-xlight);
+		}
+	}
+}
+</style>
+
+<style lang="scss">
+.value-survey {
+	.el-drawer {
+		background: var(--color-background-dark);
+
+		&__header {
+			height: 50px;
+			margin: 0;
+			padding: 18px 0 16px;
+
+			.el-dialog__close {
+				font-weight: var(--font-weight-bold);
+				color: var(--color-text-xlight);
+			}
+		}
```

Comment:
let's talk. maybe we can make these changes globally and simply update the original stylesheet
https://github.com/n8n-io/n8n/blob/master/packages/design-system/theme/src/drawer.scss

Reply:
Oki :)